src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/NewAuthModal.java

```diff
@@ -0,0 +1,53 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+/**
+ * Created by qaga on 2015-09-16.
+ */
+public class NewAuthModal {
+
+  @FindBy(css = ".auth.desktop.signin-page")
+  private WebElement newAuthModal;
+  @FindBy(css = "#loginUsername")
+  private WebElement usernameField;
+  @FindBy(css = "#loginPassword")
+  private WebElement passwordField;
+  @FindBy(css = "#loginSubmit")
+  private WebElement signInButton;
+
+  private WebDriver webDriver;
+
+  public NewAuthModal(WebDriver webdriver){
+    this.webDriver = webdriver;
```

Comment:
We decided with @ludwikkazmierczak to use webDriver name not driver.
Ludwik, is that correct and is there some bigger plan for the whole repo behind this?